<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>change_road</class>
 <widget class="QWidget" name="change_road">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>356</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>534</width>
    <height>356</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>534</width>
    <height>356</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#change_road{image: url(:/new/prefix1/bus.jpg);}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入您想要修改的道路ID：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择您想要修改的字段：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="nameButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>95</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>道路名称</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="numButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>95</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>道路编号</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>95</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>起始站台ID</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="endButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>95</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>终止站台ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入您想要修改后的内容：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="changeEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 237, 255);</string>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 237, 255);</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
